### ISSUE TITLE
Specify that spotify is included

### ISSUE THREAD

**Issue by Star-veil** [the-argus/spicetify-nix#56]
I am not sure if it's just a me problem, but at first I was certain that I needed to install the Spotify package as well, thinking this was similar to the cli tool, which obviously lead to collision issues. Fixing it is really simple, and might help others.

**Comment by eljamm** [comment]
I agree on this proposal. I also thought the same thing, and while I didn't have collisions I was surely confused when spotify didn't change even though I set up spicetify-nix correctly. Once I removed the spotify package everything worked 😁

**Comment by Razkaroth** [comment]
Yep, in my case the  system build failed because of the collisions between this and the Spotify package
